frida_tools/application.py

```diff
@@ -102,9 +104,13 @@ def store_target(option, opt_str, target_value, parser, target_type, *args, **kw
             parser.add_option("--debug", help="enable the Node.js compatible script debugger",
                 action='store_true', dest="enable_debugger", default=False)
 
+        parser.add_option("-O", "--options-file", help="text file containing additional command line options",
+                metavar="FILE", type='string', action='store')
+
         self._add_options(parser)
 
-        (options, args) = parser.parse_args()
+        real_args = self.return_real_args (parser)
```

Comment:
```suggestion
        real_args = self.return_real_args(parser)
```

Perhaps naming this something like `compute_real_args` would be clearer?

Reply:
Done